terminology.md

```diff
@@ -33,3 +34,5 @@ The Valhalla organization is comprised of several repositories each responsible
 * tiled routing - method of building a path on graph data that has been split into square cells.
 * time - the number of seconds estimated to complete a maneuver or trip, including any additional costs.
 * trip - results of an entire route, including locations, legs, and maneuvers.
+* elevation - the height above sea level at a specific location (lat,lng).
```

Comment:
should be the height with respect to sea level (can be below sea level in other words)